web/src/components/CityNotCooperatingFooter.tsx.orig

```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { CITY_NOT_COOPERATING_ROUTE, pathnameFromRouteInformation } from 'api-client/src'
+
+import buildConfig from '../constants/buildConfig'
+
+const FooterContainer = styled.div`
+  background-color: ${props => props.theme.colors.backgroundAccentColor};
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+<<<<<<< HEAD
+  border-bottom: 2px solid #b1b1b1;
+=======
+  border-bottom: 2px solid #B1B1B1;
+>>>>>>> a77968e42d7dc6205176b74dc1798a2bf95c2d01
```

Comment:
Looks like a failed merge here. I think the accepted one should be the lower case one, right?